src/test/e2e/13_find_images_test.go

```diff
@@ -51,17 +51,19 @@ func TestFindImages(t *testing.T) {
 		registry := "coolregistry.gov"
 		agentTag := "test"
 
-		stdOut, _, err := e2e.Zarf(t, "prepare", "find-images", ".", "--registry-url", registry, "--create-set", fmt.Sprintf("agent_image_tag=%s", agentTag))
+		stdOut, _, err := e2e.Zarf(t, "dev", "find-images", ".", "--registry-url", registry, "--create-set", fmt.Sprintf("agent_image_tag=%s", agentTag), "--skip-cosign")
+
 		require.NoError(t, err)
 		internalRegistryImage := fmt.Sprintf("%s/%s:%s", registry, "zarf-dev/zarf/agent", agentTag)
-		require.Contains(t, stdOut, internalRegistryImage, "registry image should be found with registry url")
-		require.Contains(t, stdOut, "busybox:latest", "Busybox image should be found as long as helm chart doesn't error")
+		require.Contains(t, stdOut, internalRegistryImage)
+		// busybox image is in git init package
+		require.Contains(t, stdOut, "busybox:latest")
 
-		path := filepath.Join("src", "test", "packages", "00-find-images-with-vars")
-		stdOut, _, err = e2e.Zarf(t, "prepare", "find-images", path, "--deploy-set", "BUSYBOX_IMAGE=busybox:earliest")
+		path := filepath.Join("src", "test", "packages", "13-find-images-with-vars")
+		stdOut, _, err = e2e.Zarf(t, "dev", "find-images", path, "--deploy-set", "PODINFO_IMAGE=cool-image.com/agent:latest", "--skip-cosign")
```

Comment:
Not sure what to think about using random domain in test. This could in theory resolve in the future.

Reply:
yeah fair, updated